tests/internal/e2e/test/test_utils/vcast_utils.ts

```diff
@@ -1230,3 +1230,69 @@ export async function selectItem(contentAssist: ContentAssist, item: string) {
 function normalizeContentAssistString(content: string): string {
   return content.replace(/⏎/g, "\n").replace(/\s+/g, " ").trim();
 }
+
+export async function getTextFromTestResultsPanel() {
+  // Open Test Results
+  await browser.keys([Key.Control, Key.Shift, "p"]);
+
+  // Get command to open Test Results
+  for (const character of "Test Results: Focus") {
+    await browser.keys(character);
+  }
+  await browser.keys(Key.Enter);
+
+  // Select the div with aria-label 'Test Results Section'
+  const testResultsPanel = await browser.$(
+    '//*[@id="workbench.panel.testResults"]'
+  );
+
+  await testResultsPanel.waitForExist();
+
+  if (!testResultsPanel) {
+    console.log("Still not found");
```

Comment:
Should this be a "harder" error?

Reply:
This was an old function and should receive the parent div from the Test Results. Did not work out. I deleted it in f823e4f71e421cd061a335b8f3421c47ee1eb035 .